cf/irsa-api-role.yaml

```diff
@@ -81,22 +80,42 @@ Resources:
             Version: "2012-10-17"
             Statement:
               - Effect: Allow
-                Action: 's3:GetObject'
+                Action: "s3:GetObject"
                 Resource:
                   - !Sub "arn:aws:s3:::worker-results-${Environment}/*"
         - PolicyName: !Sub "can-access-details-of-redis-cluster-${Environment}"
           PolicyDocument:
             Version: "2012-10-17"
             Statement:
               - Effect: Allow
-                Action: 'elasticache:DescribeReplicationGroups'
+                Action: "elasticache:DescribeReplicationGroups"
                 Resource:
                   - !Sub "arn:aws:elasticache:${AWS::Region}:${AWS::AccountId}:replicationgroup:biomage-redis-${Environment}"
         - PolicyName: !Sub "can-access-details-of-kubernetes-cluster-${Environment}"
           PolicyDocument:
             Version: "2012-10-17"
             Statement:
               - Effect: Allow
-                Action: 'eks:DescribeCluster'
+                Action: "eks:DescribeCluster"
                 Resource:
                   - !Sub "arn:aws:eks:${AWS::Region}:${AWS::AccountId}:cluster/biomage-${Environment}"
+        - PolicyName: !Sub "can-manage-state-machine-${Environment}"
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "states:CreateStateMachine"
+                  - "states:UpdateStateMachine"
+                  - "states:StartExecution"
```

Comment:
This should probably fall under the `can-manage-execution-${Environment}` policy.

Reply:
Oh ok, so I just put the two resource types (statement and execution) under one policy?

Reply:
updated in latest commit. Can you please take a look at it again?